### PR TITLE
ability to use any characters as prefix

### PR DESCRIPTION
**Pull request by TABmk**

### CODE REVIEW

**Comment by leMaik** [comment]
:smile: Nice to see someone using this library. What's the use case for customizing the prefix? Is it to recognize color codes starting wiht `&`?

**Comment by TABmk** [comment]
Almost all server plugins use `&` instead of `§` in configs now and then just replace them back. It would be great ability for example if you doing input with color text preview. You can set prefix `&` and then just copy text to your config.

File: README.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ import McText from 'mctext-react'
 | children * | `string` or `object` | | The text to display, see above. |
 | colormap | `object` | _see below_ | An alternate colormap used to map Minecraft colors to CSS colors. |
 | randomChars | `string` | `A…Za…z0…9!§$%&?#` | Characters to use for obfuscated text. |
+| prefix | `string` | `§` | Character to use for color prefix. |
```
Comment (leMaik) [inline README.md]: Maybe add ", usually `§` or `&`" to the description?

Comment (TABmk) [reply README.md]: updated

**Comment by leMaik** [comment]
Thank you, @TABmk :tada:

**Comment by leMaik** [comment]
A new version is now available on npm :rocket: 
https://github.com/TeamWertarbyte/mctext-react/releases/tag/v1.1.0

**Pull request merged**